crates/polars-core/src/chunked_array/ops/float_sorted_arg_max.rs

```diff
@@ -0,0 +1,88 @@
+//! Implementations of the ChunkAgg trait.
+use self::search_sorted::{
+    binary_search_array, slice_sorted_non_null_and_offset, SearchSortedSide,
+};
+use crate::prelude::*;
+
+macro_rules! impl_sorted_float_arg_max {
```

Comment:
Could we make this generic? I want to use macro's only for dispatch whenever possible.

Reply:
Yeap, much better. Thank you! :)